spin/dataformat-xml-dom-jakarta/pom.xml

```diff
@@ -30,10 +29,11 @@
       <groupId>org.camunda.commons</groupId>
       <artifactId>camunda-commons-utils</artifactId>
     </dependency>
-    
+
     <dependency>
       <groupId>jakarta.xml.bind</groupId>
       <artifactId>jakarta.xml.bind-api</artifactId>
+      <version>${version.jakarta.xml.bind-api}</version>
```

Comment:
❓Instead of a separate versioning, could we maybe also use the `jakarta.jakartaee-bom` here? That's already defined and used at several other places so it could be easier for future updates.

Reply:
@venetrius, I just remembered that I left a question here that wasn't answered yet.

Reply:
@danielkelemen, I forgot to link the follow up issue that will complete this update: 
https://github.com/camunda/camunda-bpm-platform/issues/4325